Tabby/components/categories/TestingMenuSelect.tsx

```diff
@@ -0,0 +1,76 @@
+// all code from chat gpt
```

Comment:
This needs more of a description according to the project requirements on the class website: https://johnxu21.github.io/teaching/CS472/project/

Reply:
I deleted it it is not being used I was trying to see if I could get something to work with a drag and drop menu without using a package but will probably use something like: https://www.npmjs.com/package/react-native-draggable-flatlist?activeTab=readme

Reply:
for the two sets of reviews this is just test data for now right now there is no actual book data being stored just components or sections of pages that take the test data and display it and do actions based on it to just have how the data should be displayed and looked

Reply:
Also added some more comments to the code